### PR TITLE
Added `buildCommand` Macro

### PR DESCRIPTION
**Pull request by Mootikins**
Added macro to build `CommandInfo` struct

### CODE REVIEW

**Comment by Toyz** [comment]
This fixes the todo left in main.cpp @ThePrimeagen i recommend merging this

**Comment by Mootikins** [comment]
I'm unsure why the MacOS build failed, it said it couldn't find the symbols.
```
Scanning dependencies of target uhh-class
[ 98%] Building CXX object src/uhh/CMakeFiles/uhh-class.dir/uhh.cpp.o
[ 98%] Linking CXX shared library libuhh-class.dylib
Undefined symbols for architecture x86_64:
  "Git::printError(int)", referenced from:
      Uhh::readyGit()     in uhh.cpp.o
  "Git::clone(UhhConfig const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)", referenced from:
      Uhh::readyGit()     in uhh.cpp.o
  "Git::commit(UhhConfig const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&)", referenced from:
      Uhh::sync()     in uhh.cpp.o
  "Config::set(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)", referenced from:
      Uhh::initPreferences()      in uhh.cpp.o
  "Config::load()", referenced from:
      Uhh::readPreferences()      in uhh.cpp.o
  "Config::save()", referenced from:
      Uhh::initPreferences()      in uhh.cpp.o
  "Config::operator[](std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >)", referenced from:
      Uhh::readPreferences()      in uhh.cpp.o
ld: symbol(s) not found for architecture x86_64
collect2: error: ld returned 1 exit status
make[2]: *** [src/uhh/libuhh-class.dylib] Error 1
make[1]: *** [src/uhh/CMakeFiles/uhh-class.dir/all] Error 2
make: *** [all] Error 2
##[error]Process completed with exit code 2.
```
Should we be concerned about this?

**Comment by Toyz** [comment]
Yea theres a issues for the MacOS fixed to build failing, you can see it at issue #13

File: src/main.cpp [hunk 1]
```diff
@@ -57,44 +57,25 @@ int findFunction(Uhh& uhh, const std::string name, const std::vector<std::string
     return 0;
 }
 
+#define buildCommand(name, usage, function) {name, usage, function}
```
Comment (ThePrimeagen) [inline src/main.cpp:60]: This is the only thing I don't like.  We can fix it in a bit.  

macros, afaik, are typically all upper case to let the user know that its a macro :)

Either way.  I'll merge and change some day.  Some day when the wind blows

**Pull request merged**